### PR TITLE
Bugfix/docker wkhtml version

### LINKED ISSUE #93: Fix version of wkhtml from Dockerfile
Current version of wkhtml in the Dockerfile is not valid anymore, update it to use version from Github.
```
RUN cd /root \
    && wget https://github.com/wkhtmltopdf/packaging/releases/download/0.12.6-1/wkhtmltox_0.12.6-1.stretch_amd64.deb --no-verbose \
    && dpkg -i wkhtmltox_0.12.6-1.stretch_amd64.deb
```

### PR DESCRIPTION
**Pull request by ejwaibel**
Closes #93

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -20,14 +20,12 @@ RUN apt-get update \
...
+    && dpkg -i wkhtmltox_0.12.6-1.stretch_amd64.deb
 
 # Enables continously calling a command and piping the output to STDOUT, viewable via docker logs
 RUN printf '#!/bin/bash\nwhile sleep 1; do\n    "$@"\ndone' >> /usr/bin/watch-docker \
     && chmod +x /usr/bin/watch-docker
 
 COPY --from=composer /app/vendor /app/vendor
 COPY . /app
-
-RUN ln -s /app/bin/md2resume /usr/bin/md2resume
```
Comment (spawnia) [inline Dockerfile:33]: Are you sure this is no longer needed?

Comment (ejwaibel) [reply Dockerfile:33]: No, that shouldn't have been removed, thought it was a line of code I had added for testing, my bad!

**Review by spawnia — APPROVED** [review]
LGTM

**Comment by craig-davis** [comment]
@spawnia This is ready to merge?

**Pull request merged**